src/Mod/Material/App/CMakeLists.txt

```diff
@@ -103,7 +105,18 @@ endif(FREECAD_USE_PCH)
 add_library(Material SHARED ${Material_SRCS})
 target_link_libraries(Material ${Material_LIBS})
 
-SET_BIN_DIR(Material Material /Mod/Material)
+# SET_BIN_DIR(Material Material /Mod/Material)
+SET_BIN_DIR(Material Material)
 SET_PYTHON_PREFIX_SUFFIX(Material)
 
-INSTALL(TARGETS Material DESTINATION ${CMAKE_INSTALL_LIBDIR})
+# INSTALL(TARGETS Material DESTINATION ${CMAKE_INSTALL_LIBDIR})
+if(WIN32)
+    INSTALL(TARGETS Material
+        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    )
+else(WIN32)
+    INSTALL(TARGETS Material
+        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+    )
+endif(WIN32)
```

Comment:
@davesrocketshop @chennes any reason to remove the Material library from Mod/Material directory?
This causes a bug if a old version of the file is still present in that directory since Mod/Material takes precedence over /lib in Python's search path. The library should stay in Mod/Material as it happens respectively with the others modules.

Reply:
This can cause issues when the src/Gui library references the Material library. The Material folder isn't in its search path and I was unable to find a way to force it. There may well be a better way to do it but I haven't found it.

Reply:
The core system (src/Base, src/App, src/Gui) should not depend on application modules like Mod/Material https://forum.freecad.org/viewtopic.php?p=448980#p448980

Can you point out where this dependency occurs so we can think of another approach?

Reply:
App/Material.*
App/MaterialObject.*
App/MaterialPy.xml
App/MaterialPyImpl.cpp
Gui/DlgMaterialProperties.*

That's a start. I believe there are also some view providers and others that will also need to be addressed as well, and there may be more. Moving those out of App and Gui are possible but that would be significantly more disruptive.

Reply:
I'm looking in those files and I can't find any dependency on /Mod/Material.
Can you show some line of code from src/Gui or src/App where Mod/Material is necessary?

Reply:
I guess you are trying to extend the App::Material objects with the functionality provided by the Material application module.
If that is the goal, you can try creating  virtual functions in App::Material and then implement them in the Material module.
Note that the core system should not know anything about the implementations in the application modules or have a dependency on them.

Here some comments from wwmayer in a similar situation: https://forum.freecad.org/viewtopic.php?p=450724#p450724

Reply:
@davesrocketshop I will create a pull request to revert the changes in CMakeLists.txt